src/models/gateway-account-credential/Worldpay3dsFlexCredential.class.ts

```diff
@@ -1,27 +1,36 @@
+import { Worldpay3dsFlexCredentialData } from '@models/gateway-account-credential/dto/Worldpay3dsFlexCredential.dto'
+
 class Worldpay3dsFlexCredential {
-  withOrganisationalUnitId (organisationalUnitId) {
+  public organisationalUnitId: string | undefined
+  public issuer: string | undefined
+  public jwtMacKey: string | undefined
+  public exemptionEngineEnabled: boolean | undefined
+  public corporateExemptionsEnabled: boolean | undefined
+
+  withOrganisationalUnitId (organisationalUnitId: string) {
     this.organisationalUnitId = organisationalUnitId
     return this
   }
 
-  withIssuer (issuer) {
+  withIssuer (issuer: string) {
     this.issuer = issuer
     return this
   }
 
-  withJwtMacKey (jwtMacKey) {
+  // not returned by Connector
```

Comment:
?

Reply:
this comment is here to explain why JwtMacKey isn't present in the `fromJson` static constructor

Reply:
Hmm, [it is though](https://github.com/alphagov/pay-connector/blob/master/src/main/java/uk/gov/pay/connector/gatewayaccount/model/Worldpay3dsFlexCredentials.java#L19)?

Reply:
has a `@JsonIgnore` annotation, yeah